library/common/engine.cc

```diff
@@ -93,6 +93,19 @@ Engine::~Engine() {
   main_thread_.join();
 }
 
+void Engine::recordCounter(std::string elements, uint64_t count) {
```

Comment:
Can we cover this in tests?

Reply:
Adding a ticket to cover testing generally at this layer, will link here.

Reply:
@goaway don't forget to add this ticket

Reply:
Created (but it's internal).